AppSettings/AppSettings/ConnectionStringAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;

```

Comment:
Rather than create another attribute, did you consider making it a property of the AppSetting one? Like ConnectionString=true?